clouddriver-openstack/src/main/groovy/com/netflix/spinnaker/clouddriver/openstack/client/OpenstackClientProvider.groovy

```diff
@@ -527,10 +526,10 @@ abstract class OpenstackClientProvider {
    * @param timeoutMins
    * @return
    */
-  void updateStack(String region, String stackName, String stackId, ServerGroupParameters parameters) {
+  void updateStack(String region, String stackName, String stackId, String template, Map<String, String> subtemplate, ServerGroupParameters parameters) {
```

Comment:
The naming of template and subtemplate is confusing. Please either flesh out the javadoc to explain what these are.